akka-http-core/src/main/scala/akka/http/impl/engine/parsing/HttpRequestParser.scala

```diff
@@ -220,6 +223,11 @@ private[http] final class HttpRequestParser(
         }
       } else failMessageStart("Request is missing required `Host` header")
 
+    private def remoteAddressStr: String =
+      inheritedAttributes.get[HttpAttributes.RemoteAddress].map(_.address) match {
+        case Some(addr) => s" from ${addr.getHostString}"
```

Comment:
Let's also add the port if we are at it. Might be random most of the times but might still help associating it with client logs.

Reply:
Ah, started out with that in, but then thought it wouldn't be useful. I'll re-add